service/deployer/notifier/slack/slack.go

```diff
@@ -0,0 +1,165 @@
+package slack
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/nlopes/slack"
+
+	microerror "github.com/giantswarm/microkit/error"
+	micrologger "github.com/giantswarm/microkit/logger"
+
+	eventerspec "github.com/giantswarm/draughtsman/service/deployer/eventer/spec"
+	"github.com/giantswarm/draughtsman/service/deployer/notifier/spec"
+)
+
+const (
+	// goodColour is the colour to use for success Slack messages.
+	goodColour = "good"
+	// dangerColour is the colour to use for failure Slack messages.
+	dangerColour = "danger"
+
+	// titleFormat is the format for titles for Slack messages.
+	titleFormat = "%v - %v"
+	// successMessage is the message for success Slack messages.
+	successMessage = "Successfully deployed"
```

Comment:
IMO it would be nice to see what has been deployed and where. E.g. the API was mentioned in the title and then there was some hash. I was wondering what this hash represented and where the API was actually deployed.